examples/humiofilteralert.yaml

```
@@ -0,0 +1,27 @@
+apiVersion: core.humio.com/v1alpha1
+kind: HumioFilterAlert
+metadata:
+  name: example-alert-filter-managed
+spec:
+  managedClusterName: example-humiocluster
+  name: example-filter-alert
+  viewName: humio
+  queryString: "#repo = humio | error = true"
+  enabled: true
+  description: Error counts
+  actions:
+    - example-email-action
+---
+apiVersion: core.humio.com/v1alpha1
+kind: HumioAction
```

```suggestion
kind: HumioFilterAlert
```

Maybe we could also get https://github.com/humio/humio-operator/blob/master/examples/humioalert.yaml#L21 updated to say `kind: HumioAlert`? I assume that's the file that was the base of this new file, but that base file doesn't seem right.

---

Yes, will fix.